libs/spark/stories/dropdown.stories.tsx

```diff
@@ -22,43 +26,87 @@ export default {
       control: 'select',
       options: ['bottom-left', 'bottom-right', 'top-left', 'top-right'],
     },
+    component: {
+      options: Object.keys(components),
+      mapping: components,
+      control: {
+        type: 'select',
+      },
+    },
   },
   args: {
     placement: 'bottom-left',
+    component: 'Button',
   },
 } as Meta;
 
-const ConfigurableTemplate: Story = ({ disabled, open, placement }) => {
+const ConfigurableTemplate: Story = ({
+  disabled,
+  open,
+  placement,
+  component,
+}) => {
   const optionalMenuProps: { open?: boolean } = {};
   if (open !== undefined) optionalMenuProps.open = open;
 
+  const position = getPosition(placement);
+
   return (
-    <Box mx={14} my={20}>
-      <DropdownContext>
-        <DropdownButton endIcon={<ChevronDown />} disabled={disabled}>
-          Label
-        </DropdownButton>
-        <DropdownMenu placement={placement} {...optionalMenuProps}>
-          <MenuItem>
-            <ListItemIcon>
-              <UserDuotone />
-            </ListItemIcon>
-            <ListItemText primary="Profile" />
-          </MenuItem>
-          <MenuItem>
-            <ListItemText primary="Resources" />
-          </MenuItem>
-          <MenuItem disabled>
-            <ListItemText primary="Menu item" />
-          </MenuItem>
-          <Divider />
-          <MenuItem>
-            <ListItemText primary="Sign out" />
-          </MenuItem>
-        </DropdownMenu>
-      </DropdownContext>
+    <Box position="relative" height={182 + 48 + 8} width={232}>
```

Comment:
What's happening here with the height of 182 + 48 + 8?

Reply:
height of menu + height of (largest possible) anchor element + space between anchor and menu

Whereas the width is always the menu's width.

Reply:
Got it, thanks, William!